sprints/20200526-support-worker-reregistration/sprint-retrospective.md

```diff
@@ -0,0 +1,60 @@
+# Sprint retrospective - Support worker re-registration
```

Comment:
Should we add the start date and end date here? I know the start (or end?) date is in the directory name, but would it also be useful in the document header too?